docs/volunteering/tasks.md

```diff
@@ -0,0 +1,55 @@
+# Tasks
+
+We record tasks which need doing at https://github.com/srobo/tasks/issues.
+
+If you spot a task which you'd like to do, please `@mention` the person who
+created the task to let them know that you're interested. If you're able to
+self-assign the task (for technical reasons you won't initially be able to do
+this), please also do that.
+
+<!--- Review question: is the original author the right person to ping here? --->
+
+This lets the original author know that you're interested, meaning that they can
+pass on any additional information which might be needed to complete the task.
+Alternatively, if they're unsure, they can likely suggest someone else who would
+be able to advise you.
+
+Assignment also acts as a way for others to know that you're working on
+something, so that they don't need to duplicate your efforts!
+
+If you see a task which you think someone else would be able to take on (perhaps
+because they expressed a particular interest in doing so, or are already doing
+something very similar), please `@mention` them on the task asking if they'd be
+interested.
+
+!!! note
+
+    Please do _not_ assign other people to tasks unless they have specifically
+    said that they are doing the task.
+
+If you're asked whether you can take on a task, there is no obligation to do so.
+However please do ensure that you respond in a timely fashion so it's clear you
+won't be able to take on a task.
+
+If you've taken on a task and find that you no longer have the time or energy to
+complete it, that's fine! However please do post on the task mentioning that
+you're no longer able to complete it and remove your assignment from the task.
```

Comment:
NB: Most volunteers cannot currently unassign themselves.

Reply:
This is false. Anyone who can be assigned an issue also has the permission to unassign themselves, and change anyone else who's assigned.

(At time of writing, this applies to *everyone* with access to the repo, either through the org or as a collaborator. If it happens in future, it'll be because they've been added since and were incorrectly setup)
(If i'm wrong, send me a username of someone who has the issue now and i'll look into it)

Reply:
If the text reflects the intended state, then that's fine. I'll have a chat with the people who've mentioned this to me. Even if it is still the case, this is not the place to discuss it.